### PR TITLE
 Added Persian translations

### PR DESCRIPTION
**Pull request by xmha97**

### CODE REVIEW

**Comment by xmha97** [comment]
What is the problem? I tried to translate all the phrases correctly.

**Comment by z-huang** [comment]
Please fix the translation according to the build log. You should pass the build before I merge it.

**Comment by xmha97** [comment]
> Please fix the translation according to the build log. You should pass the build before I merge it.

OK, now you can merge my PR.

**Comment by xmha97** [comment]
@z-huang No matter what I tried, I could not find the problem. Please do it yourself. 🙏

File: app/src/main/res/values-fa-rIR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    <string name="sort_by_year">سال</string>
+    <string name="sort_by_song_count">تعداد آهنگ</string>
+    <string name="sort_by_length">طول</string>
+
+    <!-- Snackbar -->
+    <plurals name="snackbar_delete_song">
+        <item quantity="one">%d آهنگ حذف‌شده‌است.</item>
+        <item quantity="other">%d آهنگ‌ها حذف‌شده‌است.</item>
+    </plurals>
+    <plurals name="n_selected">
```
[the remark this answers is not in the document]
Comment (xmha97) [reply app/src/main/res/values-fa-rIR/strings.xml:177]: Sometimes in the Persian language, `quantity one` = `quantity other`. for example:
"1 book, 5 books" is correct in English but
"1 book, 5 book" is correct in Persian. 😁

[the remark this answers is not in the document]
Comment (z-huang) [reply app/src/main/res/values-fa-rIR/strings.xml:177]: Then please add `<item quantity="other">%d انتخاب‌شده</item>` after this line.

**Review by z-huang — APPROVED** [review]

**Pull request merged**